src/simple-cache-client.ts

```diff
@@ -162,6 +163,21 @@ export class SimpleCacheClient {
     return await this.controlClient.listCaches(nextToken);
   }
 
+  /**
+   * Fetch the entire dictionary from the cache.
+   * @param {string} cacheName - Name of the cache to perform the lookup in.
+   * @param {string} dictionaryName - The dictionary to fetch.
+   * @returns Promise<DictionaryFetch.Response> - Promise containing the result of the fetch operation and the associated dictionary.
+   * @memberof SimpleCacheClient
```

Comment:
I think [types have to be in brackets](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns).

```suggestion
   * @returns {Promise<DictionaryFetch.Response>} - Promise containing the result of the fetch operation and the associated dictionary.
   * @memberof {SimpleCacheClient}
```